Makefile

```diff
@@ -94,176 +93,32 @@ test:
-int: TESTARGS= $(VERBOSE_FLAG) -timeout 3m
-int:
-	$(DOCKER_BUILD) 'CGO_ENABLED=0 $(INT_TEST)'
-
-lint:
-	$(DOCKER_BUILD) '$(LINT)'
-
-vet:
-	$(DOCKER_BUILD) 'CGO_ENABLED=0 $(VET)'
-
-pre:
-	wget https://github.com/estesp/manifest-tool/releases/download/v1.0.1/manifest-tool-linux-amd64 \
-	  -O manifest-tool && \
-	 chmod +x ./manifest-tool
-	echo $(DOCKERHUB_TOKEN) | docker login --username sonobuoybot --password-stdin
-
-# TODO: Make it easy to build single container for a specific arch
-build_container:
-	$(DOCKER) build \
-       -t $(REGISTRY)/$(TARGET):$(IMAGE_VERSION) \
-       -t $(REGISTRY)/$(TARGET):$(IMAGE_TAG) \
-       -t $(REGISTRY)/$(TARGET):$(IMAGE_BRANCH) \
-       -t $(REGISTRY)/$(TARGET):$(GIT_REF_SHORT) \
-       -f $(DOCKERFILE) \
-		.
-
-linux_containers: build/linux/arm64/sonobuoy build/linux/amd64/sonobuoy
-	for arch in $(LINUX_ARCH); do \
-		if [ $$arch = amd64 ]; then \
-			sed -e 's|BASEIMAGE|$(AMD_IMAGE)|g' \
-			-e 's|CMD1||g' \
-			-e 's|BINARY|build/linux/amd64/sonobuoy|g' Dockerfile > Dockerfile-$$arch; \
-			$(MAKE) build_container DOCKERFILE=Dockerfile-$$arch; \
-			$(MAKE) build_container DOCKERFILE="Dockerfile-$$arch" TARGET="sonobuoy-$$arch"; \
-	elif [ $$arch = arm64 ]; then \
-			sed -e 's|BASEIMAGE|$(ARM_IMAGE)|g' \
-			-e 's|CMD1||g' \
-			-e 's|BINARY|build/linux/arm64/sonobuoy|g' Dockerfile > Dockerfile-$$arch; \
-			$(MAKE) build_container DOCKERFILE="Dockerfile-$$arch" TARGET="sonobuoy-$$arch"; \
-		else \
-			echo "Linux ARCH unknown"; \
-        fi \
-	done
-
-windows_containers: build/windows/amd64/sonobuoy.exe
-	@echo container_win
-	for arch in $(WIN_ARCH); do \
-		if [ $$arch = amd64 ]; then \
-			sed -e 's|BASEIMAGE|$(WIN_IMAGE)|g' \
-			-e 's|BINARY|build/windows/amd64/sonobuoy.exe|g' DockerfileWindows > DockerfileWindows-$$arch; \
-			$(MAKE) build_container DOCKERFILE=DockerfileWindows-$$arch; \
-			$(MAKE) build_container DOCKERFILE="DockerfileWindows-$$arch" TARGET="sonobuoy-win-$$arch"; \
-		else \
-			echo "Windows ARCH unknown"; \
-        fi \
-	done
-
 build_sonobuoy:
 	$(DOCKER_BUILD) '$(GO_BUILD)'
 
-build/linux/arm64/sonobuoy:
-	echo Building: linux/arm64
-	mkdir -p build/linux/arm64
-	$(MAKE) build_sonobuoy GO_SYSTEM_FLAGS="GOOS=linux GOARCH=arm64" BINARY=$@
-
-build/linux/amd64/sonobuoy:
-	echo Building: linux/amd64
-	mkdir -p build/linux/amd64
-	$(MAKE) build_sonobuoy GO_SYSTEM_FLAGS="GOOS=linux GOARCH=amd64" BINARY=$@
+# Integration tests
+int: ARTIFACTS_DIR=/tmp/artifacts KUBECONFIG=$(KUBECONFIG) SONOBUOY_CLI=$(SONOBUOY_CLI)
+int: TESTARGS=$(VERBOSE_FLAG) -timeout 3m
+int: native
+	KO_DOCKER_REPO=ko.local \
+	KIND_CLUSTER_NAME=$(KIND_CLUSTER) \
+	$(KO) publish --platform=linux/amd64 --tags $(GIT_VERSION) --base-import-paths github.com/vmware-tanzu/sonobuoy
+	docker tag ko.local/sonobuoy:$(GIT_VERSION) sonobuoy/sonobuoy:$(GIT_VERSION)
+	kind load docker-image sonobuoy/sonobuoy:$(GIT_VERSION)
```

Comment:
so we kinda need to do this hacky thing instead of ko directly deploying to kind because it's harder to modify the integration test such that they will be pointing to `ko.local/sonobuoy` instead of `sonobuoy/sonobuoy` since I don't think we necessarily want users to run ko?